Integración con Tailwind y HomePage con Tailwind!

evertms opened the pull request:
Implementada una NavBar desarrollada con TailwindCSS. Todavía están medio rústicas las formas, tengo que crear las otras tres páginas.
![image](https://github.com/user-attachments/assets/a912bf91-34b0-4465-8d8a-94850f056f20)

evertms committed:
Integración con Tailwind y HomePage con Tailwind!

Todavía están medio rústicas las formas, tengo que crear las páginas

evertms committed:
Update readmes

RaidenMT commented on OrderNow-React/src/pages/Home.jsx:
El archivo está vacío, si no se usará, eliminar

RaidenMT replied:
Sigue sin utilizarse, está vacío este archivo

evertms committed:
Usando Home.jsx!

evertms committed:
Uso de patrón de diseño Layout

Además de ello, corrección de ciertos errores además del mostrado por el ing. Raiden

RaidenMT reviewed (approved):
Hay mejoras pero las hacemos en el siguiente PR.
